cpp/src/join/join_kernels.cuh

```diff
@@ -94,6 +94,23 @@ __global__ void build_hash_table(multimap_type multi_map,
     }
 }
 
+template <typename Map>
+__global__ void build_hash_table(Map map, size_type num_rows) {
```

Comment:
This looks like it could just be a `Thrust` algorithm instead of a kernel.

Reply:
Made this a thrust algorithm.